README.md

```diff
@@ -1,45 +1,60 @@
 # @freenow/react-polygon-editor
 
 ### Table of contents
-* About
-* Getting started
-* Components
-* How to run locally
-* Contribution
 
+-   About
+-   Getting started
+-   Components
+-   How to run locally
+-   Contribution
 
 ### About
+
 React Polygon Editor provides react components for displaying and editing polygons.
 We use leaflet for rendering maps. And typescript to provide a strongly typed interface.
 
 ### Getting started
 
 First install @freenow/react-polygon-editor:
+
 ```bash
 npm i -S @freenow/react-polygon-editor
 ```
 
 Make sure you have also installed all peer dependencies. Have a look at [package.json](package.json) for more information.
 
+```bash
+npm i -S react react-dom react-leaflet leaflet styled-components
+```
+
 inject the leaflet css style.
+
 ```typescript
 Import 'leaflet/dist/leaflet.css';
 ```
+
 You can also link the css style from a CDN in your index.html
+
 ```html
 <link rel="stylesheet" href="https://unpkg.com/leaflet@1.4.0/dist/leaflet.css" />
-``` 
+```
 
 ### Components
 
 ##### PolygonDraw
+
 **Props**
-* **polygon**: *Coordinate[]*
-* **boundary**?: *Coordinate[]*
-* **initialCenter**?: *Coordinate* (The initial center will be used to localize the map on the first render if no polygon or boundary polygon were provided)
-* **initialZoom**?: *number* (The initial zoom will be used to localize the map on the first render if no polygon or boundary polygon were provided)
-* **editable**?: *boolean* (Allows enabling and disabling polygon editing. **Default value: true**)
-* **onChange**?: *(polygonCoordinates: Coordinate[], isValid: boolean) => void*
+
+-   **polygon**: _Coordinate[] | Coordinate[][]_ (Single or list of polygons to render)
```

Comment:
I think having 2 competing props for the same thing would make it more complicated in my opinion.

If we change the prop we should name it something ambiguous like `coordinates` or even more generic `points`. That would be less confusing IMO while still be able to accept one or multiple polygons which is how leaflet does it.

The way bigger problem I see with the `onChange` callback. I just noticed that this PR would be a breaking change because we only return the currently active polygon and therefore making the component hard to use as a controlled component.

I'm not sure what to do about this though because returning `isValid` for the entire list also feels "wrong"? Especially because the validator is part of the public API so the user could validate the polygons themselves. I'm not sure why we added this.